desktop/index.md

```diff
@@ -0,0 +1,58 @@
+---
+description: Docker Desktop overview
+keywords: Desktop, Docker, GUI, run, docker, local, machine
+title: Docker Desktop overview
+toc_min: 1
+toc_max: 2
+---
+
+Docker Desktop is an easy-to-install application for your Mac or Windows environment that enables you to build and share containerized applications and microservices. Docker Desktop includes [Docker Engine](../engine/), Docker CLI client, [Docker Compose](../compose/), [Notary](../notary/getting_started/), [Kubernetes](https://github.com/kubernetes/kubernetes/), and [Credential Helper](https://github.com/docker/docker-credential-helpers/).
+
+Docker Desktop works with your choice of development tools and languages and gives you access to a vast library of certified images and templates in Docker Hub. This enables development teams to extend their environment to rapidly auto-build, continuously integrate and collaborate using a secure repository.
+
+Some of the key features of Docker Desktop include:
+
+* Ability to containerize and share any application on any cloud platform, in multiple languages and frameworks
+* Easy installation and setup of a complete Docker development environment
+* Includes the latest version of Kubernetes
+* Automatic updates with our preview channel - Desktop Edge and Stable versions
```

Comment:
Not sure what this line is trying to say.. :D Maybe just leave it as 'Automatic updates - keeps you up to date and secure'